ccx_keys/locator.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 """ Locator module. """
+from __future__ import unicode_literals
```

Comment:
Not sure if this is the right way to fix this. pycodestyle otherwise complains about line 29 (now 31):
```
************* Module ccx_keys.locator
C: 29, 0: Human-readable format strings should be unicode (unicode-format-string)
```

Reply:
One option is to kill Python 2.7 at the same time.  Same PR and commit would be fine with me, but also fine if you want to skip that for now.

Reply:
It was extremely tempting, but I'm already several yaks deep. :-D